Add oauth_console authentication to Starburst/Trino (#4588)
dbt-labs/docs.getdbt.com: website/docs/docs/core/connect-data-platform/trino-setup.md

      host: bunbundersders.trino.galaxy-dev.io
      catalog: dbt_target
      schema: dataders
      port: 443
```

</TabItem>

<TabItem value="oauth_console">

The only authentication parameter to set for OAuth 2.0 is `method: oauth_console`. If you're using Starburst Enterprise or Starburst Galaxy, you must enable OAuth 2.0 in Starburst before you can use this authentication method.

For more information, refer to both [OAuth 2.0 authentication](https://trino.io/docs/current/security/oauth2.html) in the Trino docs and the [README](https://github.com/trinodb/trino-python-client#oauth2-authentication) for the Trino Python client.

> The only difference between `oauth_console` and `oauth` is that in the latter a browser is automatically opened with authentication URL and in `oauth_console` URL is printed to the console.

Reviewer: ```suggestion
The only difference between `oauth_console` and `oauth` is:
- `oauth` &mdash; An authentication URL automatically opens in a browser.
- `oauth_console` &mdash; A URL is printed to the console.
```
